day_7/10_little_monkeys.rb

```diff
@@ -0,0 +1,34 @@
+num = {
+  1 => 'One',
+  2 => 'Two',
+  3 => 'Three',
+  4 => 'Four',
+  5 => 'Five',
+  6 => 'Six',
+  7 => 'Seven',
+  8 => 'Eight',
+  9 => 'Nine',
+  10 => 'Ten'
+}
+
+num.reverse_each do |x, y|
```

Comment:
What are x and y referring to here? Can you think of more descriptive names?

Reply:
Hi!

x: key
y: value 

key/value would be better variable names (I have updated the file accordingly). monkey_int/monkey_string could work as well!